### DISCUSSION TITLE
Connect to 127.0.0.1 or this devices local address.. to continue

### DISCUSSION THREAD

**Discussion by Pchelekkk** [bedrock-tool/bedrocktool#195] [Q&A] [answered]
Hello, After trying to download a world, it says "Listening, connect to 127.0.0.1 or this devices local address in the minecraft bedrock client to continue. What should I do here? I try to connect to a server with this Ip but it doesn't exist...

**Comment by GalacticInterfacify** [comment, marked as answer]
https://doc.pmmp.io/en/rtfd/faq/connecting/win10localhostcantconnect.html

**Reply by Pchelekkk** [reply]
Worked, Thanks a lot!
